### PR TITLE
Add reference to slides and presentation.

### PR DESCRIPTION
**Pull request by pashu123**

### CODE REVIEW

**Comment by pashu123** [comment]
@silvasean Should I add all the references to slides and presentations from the wiki over here?

**Comment by silvasean** [comment]
Let's add the MLIR ODM one, since that fits in well with the "architecture". The other two don't fit in very well.

**Review by silvasean — APPROVED** [review]

File: docs/architecture.md [hunk 1]
```diff
@@ -441,3 +441,7 @@ memory, doesn't support any error handling, performs no optimizations, and
 probably a bunch of other horrible things. We are patiently awaiting for the
 upstream MLIR community to produce a viable end-to-end flow with better
 characteristics.
+
+### Presentations and Talks
+
+* 2022-08-20: Overview of torch-mlir passes. ([recording](https://www.youtube.com/watch?v=ZpwlVxsD9_U&t=2374s) and [slides](https://drive.google.com/file/d/1ZSlk1HGttRuVhJSxtP6spWt_hxClit2T/view))
```
Comment (silvasean) [inline docs/architecture.md:447]: nit: torch-mlir -> Torch-MLIR

**Comment by silvasean** [comment]
LGTM!

**Pull request merged**